src/panels/lovelace/cards/hui-statistics-graph-card.ts

```diff
@@ -167,12 +172,21 @@ export class HuiStatisticsGraphCard extends LitElement implements LovelaceCard {
         1000 * 60 * 60 * (24 * (this._config!.days_to_show || 30) + 1)
     );
     try {
+      const metadata = await getStatisticMetadata(this.hass!, this._entities);
```

Comment:
This means now fetch metadata both here and in `statistics-chart`.
Maybe we should just fetch it here and pass it to the `statistics-chart`?